front/pages/api/w/[wId]/use/actions/[action]/index.ts

```diff
@@ -160,24 +172,35 @@ async function handler(
 
       const reader = apiRes.body.getReader();
 
+      const parser = createParser((event) => {
```

Comment:
Humble suggestion on readability: creating a parser + parser.feed in the code makes it hard to understand that it's error checking code (non-standard imho)
quickest suggestion: rename `parser` to `errorDetector` or smth like that
other option: const errorOnAction = fn... then if (errorOnAction(value)) logActionError

Reply:
the idea behind is "understandable quickly on first read"

Reply:
wilco :+1: